### PR TITLE
Implement purge and save the codeformatting

### PR DESCRIPTION
**Pull request by rotwurstesser**

### CODE REVIEW

File: frontend/postcss.config.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+
+  // Include any special characters you're using in this regular expression
+  defaultExtractor: content => content.match(/[\w-/:]+(?<!:)/g) || [],
+});
+
 module.exports = {
   plugins: [
     require('postcss-import')(),
     require('tailwindcss')(),
+    require('autoprefixer'),
```
Comment (ingvijonasson) [inline frontend/postcss.config.js]: We do not need `autoprefixer` it is already in `postcss-preset-env`

File: frontend/postcss.config.js [hunk 2]
```diff
@@ -1,7 +1,32 @@
+// all selectors in this array will not be purged
+let selectorWhiteListArray = [];
+// element ui
+selectorWhiteListArray.push(/^el-/);
```
Comment (JonasNiestroj) [inline frontend/postcss.config.js]: Why do we need to whitelist element ui? We dont use element ui

File: frontend/postcss.config.js [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+selectorWhiteListArray.push(/^token/);
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  // Specify the paths to all of the template files in your project
+  content: [
+    './src/**/*.html',
+    './src/**/*.vue',
+    './src/**/*.postcss',
+    // etc.
+  ],
+  // survive element ui, survive!
```
Comment (JonasNiestroj) [inline frontend/postcss.config.js]: I think its survives if we dont use it 😉

File: frontend/postcss.config.js (hunk 1 above)
Comment (rotwurstesser) [reply frontend/postcss.config.js]: ok removed

File: frontend/postcss.config.js (hunk 2 above)
Comment (rotwurstesser) [reply frontend/postcss.config.js]: removed

File: frontend/postcss.config.js (hunk 3 above)
Comment (rotwurstesser) [reply frontend/postcss.config.js]: yep xD

**Pull request merged**